package.json

```diff
@@ -52,76 +52,76 @@
     "upgrade-install": "./upgrade-install.sh"
   },
   "dependencies": {
-    "@fastify/helmet": "9.1.0",
-    "@fastify/multipart": "7.1.0",
-    "@fastify/static": "6.5.0",
-    "@fastify/swagger": "7.4.1",
+    "@fastify/helmet": "11.1.1",
+    "@fastify/multipart": "8.0.0",
+    "@fastify/static": "6.11.2",
+    "@fastify/swagger": "8.11.0",
     "@homebridge/node-pty-prebuilt-multiarch": "0.10.1-pre.6",
```

Comment:
This not ready for release yet.  The NPM package does not contain any of the prebuilt modules

Reply:
Package has been updated to v0.11.1